src/Illuminate/Cache/RedisStoreIgbinary.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Illuminate\Cache;
+
+use Illuminate\Contracts\Redis\Factory as Redis;
+
+class RedisStoreIgbinary extends RedisStore
+{
+    /**
+     * Create a new Redis store.
+     *
+     * @param  \Illuminate\Contracts\Redis\Factory  $redis
+     * @param  string  $prefix
+     * @param  string  $connection
+     * @return void
+     */
+    public function __construct(Redis $redis, $prefix = '', $connection = 'default')
+    {
+        if (
+            ! function_exists('igbinary_serialize') ||
+            ! function_exists('igbinary_unserialize')
+        ) {
+            throw new \InvalidArgumentException('Igbinary extension not found');
```

Comment:
Please add the extension as a recommendation to the cache composer.json and also the main composer.json file.

Reply:
> Please add the extension as a recommendation to the cache composer.json and also the main composer.json file.

https://github.com/maxm86545/framework/commit/ce39cc17b419561913825431a5364451d6d97030